### PR TITLE
Hooked up order confirmation to db and passed down storage services

### PR DESCRIPTION
**Pull request by rseln**

### CODE REVIEW

File: app/src/main/java/com/example/munchbox/ui/MealSelectionScreen.kt [hunk 1]
```diff
@@ -42,6 +44,8 @@ fun MealSelectionScreen(
     /**
      * State Variables
      */
+    val orderService = StorageServices(FirebaseFirestore.getInstance()).orderService()
```
Comment (rseln) [inline app/src/main/java/com/example/munchbox/ui/MealSelectionScreen.kt:47]: @Seanlebon this is what i mean by having to get the instance again

File: app/src/main/java/com/example/munchbox/ui/components/OrderSummaries.kt [hunk 2]
```diff
@@ -19,10 +19,14 @@ import com.example.munchbox.controller.DayOfWeek
 import com.example.munchbox.controller.Meal
 import com.example.munchbox.data.DataSource
 import com.example.munchbox.data.OrderUiState
+import com.example.munchbox.data.StorageServices
 import com.example.munchbox.ui.OrderViewModel
+import com.google.firebase.firestore.FirebaseFirestore
 
 @Composable
 fun OrderSummaries(order: OrderUiState, modifier: Modifier = Modifier){
+    val orderService = StorageServices(FirebaseFirestore.getInstance()).orderService()
```
Comment (rseln) [inline app/src/main/java/com/example/munchbox/ui/components/OrderSummaries.kt:28]: here too @Seanlebon

Comment (rseln) [reply app/src/main/java/com/example/munchbox/ui/components/OrderSummaries.kt:28]: We can probably pass it down once we set up everything else to be able to take in the service (aka once we restructure the code to user the DB)

File: app/src/main/java/com/example/munchbox/ui/MealSelectionScreen.kt (hunk 1 above)
Comment (rseln) [reply app/src/main/java/com/example/munchbox/ui/MealSelectionScreen.kt:47]: We can probably pass it down once we set up everything else to be able to take in the service (aka once we restructure the code to user the DB)

**Review by seanchen0818 — APPROVED** [review]

**Pull request merged**